components/mdns/mdns.c

```diff
@@ -7521,6 +7521,21 @@ static mdns_browse_t *_mdns_browse_find_from(mdns_browse_t *b, mdns_name_t *name
     return b;
 }
 
+static bool is_txt_item_exist(mdns_txt_item_t txt, uint8_t txt_value_len, mdns_txt_item_t *txt_list, uint8_t *txt_value_len_list, size_t txt_count)
```

Comment:
nitpick, suggest we give it a better name...
```suggestion
static bool is_txt_item_in_list(mdns_txt_item_t txt, uint8_t txt_value_len, mdns_txt_item_t *txt_list, uint8_t *txt_value_len_list, size_t txt_count)
```

Reply:
It's a great suggestion. Thanks.